addons/block_code/ui/blocks/block/block.gd

```diff
@@ -189,6 +189,10 @@ func _to_string():
 	return "<{block_class}:{block_name}#{rid}>".format({"block_name": definition.name if definition else "", "block_class": get_block_class(), "rid": get_instance_id()})
 
 
+func _get_tooltip(at_position: Vector2) -> String:
+	return definition.description
```

Comment:
Unlikely, but I think to be safe you should check that `definition` is defined like in `_to_string`:

```suggestion
	return definition.description if definition else ""
```

Reply:
Good point, I applied that change.